site/content/en/latest/user/security/oidc.md

```diff
@@ -16,13 +16,42 @@ This instantiated resource can be linked to a [Gateway][Gateway] and [HTTPRoute]
 Follow the steps from the [Quickstart](../../quickstart) guide to install Envoy Gateway and the example manifest.
```

Comment:
split this change to a separate PR, with a more suitable title?

Reply:
This file has been modifed by mistake, I'll remove it.